apps/chat-e2e/src/tests/sharedFilesAttachments.test.ts

```diff
@@ -0,0 +1,477 @@
+        );
+        await attachedFilesAssertion.assertEntityArrowIconColor(
+          specialCharsImageEntity,
+          Colors.controlsBackgroundAccent,
+        );
+        await attachFilesModal.closeButton.click();
+      },
+    );
+
+    await dialSharedWithMeTest.step(
+      'Prepare conversation to share of the user 2',
+      async () => {
+        additionalShareUserConversationData.resetData();
+        const conversationToShare =
+          additionalShareUserConversationData.prepareEmptyConversation(
+            ModelIds.GPT_4_O,
+          );
+        await additionalShareUserDataInjector.createConversations([
+          conversationToShare,
+        ]);
+        await additionalShareUserLocalStorageManager.setSelectedConversation(
+          conversationToShare,
+        );
+      },
+    );
+
+    await dialSharedWithMeTest.step(
+      'By user2 create a conversation with attachments from Shared with me section in Manage attachments',
+      async () => {
+        const newRequest = GeneratorUtil.randomString(10);
+        await additionalShareUserDialHomePage.openHomePage();
+        await additionalShareUserDialHomePage.waitForPageLoaded({});
+        await additionalShareUserSendMessage.attachmentMenuTrigger.click();
+
+        await additionalShareUserAttachmentDropdownMenu.selectMenuOption(
+          UploadMenuOptions.attachUploadedFiles,
+        );
+
+        await additionalShareUserAttachFilesModal.checkAttachedFile(
+          Attachment.specialSymbolsName,
+          FileModalSection.SharedWithMe,
+        );
+        await additionalShareUserAttachFilesModal.attachFiles();
+        await additionalShareUserDialHomePage.mockChatTextResponse(MockedChatApiResponseBodies.simpleTextBody);
+        await additionalShareUserChat.sendRequestWithButton(newRequest);
+        await additionalShareUserChat.waitForResponse();
+        await additionalShareUserConversations.openEntityDropdownMenu(
+          newRequest,
+        );
+        await additionalShareUserConversationDropdownMenu.selectMenuOption(
+          MenuOptions.share,
+        );
+        const errorMessage =
+          await additionalShareUserErrorToast.getElementContent();
+        await additionalShareUserShareErrorToastAssertion.assertSharingWithAttachmentNotFromAllFilesFailed(
+          errorMessage,
+        ); //TODO close the toast
+      },
+    );
+
+    await conversations.selectConversation(conversationWithTwoResponses.name);
+    for (const action of ['rename', 'model change', 'delete']) {
+      await dialTest.step(`User1 ${action}s the shared chat`, async () => {
+        switch (action) {
+          case 'rename':
+            await conversations.openEntityDropdownMenu(
+              conversationWithTwoResponses.name,
+            );
+            conversationWithTwoResponses.name = GeneratorUtil.randomString(10);
+            await conversationDropdownMenu.selectMenuOption(MenuOptions.rename);
+            await conversations
```

Comment:
there is one-line method to rename conversation:

`await conversations.editConversationNameWithTick(conversationWithTwoResponses.name);`

Reply:
done